backend/src/test/java/gov/cdc/usds/simplereport/db/repository/TestEventRepositoryTest.java

```diff
@@ -45,5 +45,8 @@ public void testLatestTestEventForPerson() {
         flush();
         TestEvent found = _repo.findFirst1ByPatientOrderByCreatedAtDesc(patient);
         assertEquals(second.getResult(), TestResult.UNDETERMINED);
+
+        List<TestEvent> found2 = _repo.findAllByCreatedAtInstant("2020-01-01T16:13:15.448000Z");
+        // assertNotEquals(2, found2.size());
```

Comment:
Erp.

Reply:
I figure checking in the half test would shame me into going back and finishing writing the test sooner. I should include my signature and a todo comment just motivate myself more.

Reply:
Fixed by making `assertEquals`... clean unit tests seem to work, but if the database already has fields it fails. Not sure the best way to do testing like this? Suggestions? Maybe run it once, get the count, add a couple entries, and then check that the count delta is correct?